client/rpc/src/eth/cache/mod.rs

```diff
@@ -288,120 +287,6 @@ where
 	BE: Backend<B> + 'static,
 	BE::State: StateBackend<BlakeTwo256>,
 {
-	/// Task that caches at which substrate hash a new EthereumStorageSchema was inserted in the Runtime Storage.
-	pub async fn ethereum_schema_cache_task(client: Arc<C>, backend: Arc<fc_db::Backend<B>>) {
-		if let Ok(None) = frontier_backend_client::load_cached_schema::<B>(backend.as_ref()) {
```

Comment:
Should we remove the `EthereumStorageSchema` cache in DB META column? @tgmichel

Reply:
I left it there on purpose because I plan to get back to this at some point. It's just a single key, I thinkn it doesn't hurt.